crates/ln-dlc-storage/src/lib.rs

```diff
@@ -439,7 +443,7 @@ impl<K: DlcStoreProvider> dlc_manager::Storage for DlcStorageProvider<K> {
         self.store
             .write(
                 CHAIN_MONITOR,
-                "chain_monitor".to_string().into_bytes(),
+                CHAIN_MONITOR_KEY.to_string().into_bytes(),
```

Comment:
🙈 ouch I've must have missed that, when reverting to the numbers for backwards compatibility. Nice catch! 💪